### PR TITLE
requirements: Pin selenium version to 4.10.x

### PR DESCRIPTION
**Pull request by orfeas-k**
Pin Selenium version to `<4.11` in charm's integration tests because upgrading to `4.11.2` results in the following issue https://github.com/canonical/dex-auth-operator/issues/151. 

Related runs 
https://github.com/canonical/dex-auth-operator/actions/runs/5763096636/job/15669866245?pr=132#step:5:2253
https://github.com/canonical/dex-auth-operator/actions/runs/5763096636/job/15627005664?pr=132#step:5:2066

### CODE REVIEW

File: requirements-integration.in [hunk 1]
```diff
@@ -6,6 +6,7 @@ lightkube
 pytest-operator
 pyyaml
 requests
-selenium
+# Pinning to 4.10.x because upgrading to 4.11.2 results in chrome version compatibility issue. See https://github.com/canonical/dex-auth-operator/issues/151.
+selenium==4.10
```
Comment (DnPlas) [inline requirements-integration.in:10]: Let's keep this a bit loose on the lower bound in case other packages (iff there are others) require selenium less than 4.10:

```suggestion
selenium<4.11
```

File: requirements-unit.txt [hunk 2]
```diff
@@ -55,7 +55,7 @@ idna==3.4
     #   anyio
     #   httpx
     #   requests
-importlib-resources==6.0.0
+importlib-resources==6.0.1
```
Comment (DnPlas) [inline requirements-unit.txt:58]: I believe this change is unrelated to this PR.

File: requirements.txt [hunk 3]
```diff
@@ -34,7 +34,7 @@ idna==3.4
     #   anyio
     #   httpx
     #   requests
-importlib-resources==6.0.0
+importlib-resources==6.0.1
```
Comment (DnPlas) [inline requirements.txt:37]: I believe this change is unrelated to this PR.

File: requirements-integration.txt [hunk 4]
```diff
@@ -166,7 +166,7 @@ pyasn1-modules==0.3.0
     # via google-auth
 pycparser==2.21
     # via cffi
-pygments==2.15.1
+pygments==2.16.1
```
Comment (DnPlas) [inline requirements-integration.txt:169]: I wonder if this bump is related to pinning selenium, it doesn't look like.

**Review by DnPlas — COMMENTED** [review]
Thanks @orfeas-k, some comments.

**Comment by orfeas-k** [comment]
@DnPlas thanks for the comments, I had actually missed those. Force pushed onto the branch.

**Review by DnPlas — APPROVED** [review]
